### DISCUSSION TITLE
Button inside MenuTrigger keeps data-pressed="true" attribute when Popover is controlled

### DISCUSSION THREAD

**Discussion by janvorwerk** [adobe/react-spectrum#5765] [Q&A] [answered]
Hello,

I just discovered React Aria and its components 🚀

I just noticed a weird behavior, when I use a button as menu trigger of a controlled popover: the button state keeps the `data-pressed="true"` after I stop pressing the button.

Something like that:

```tsx
     const [isMenuOpen, setMenuOpen] = useState(false);

...

        <MenuTrigger>
          <Button variant="primary" onPress={() => setMenuOpen(true)}>
            Filter
          </Button>
          <Popover className="min-w-[150px]" isOpen={isMenuOpen} onOpenChange={setMenuOpen}>
            <Menu
              onAction={onMenuAction}
              selectionMode="multiple"
              onSelectionChange={onFilterSelectionChange}
              selectedKeys={shownFilters}
            >
```

The behavior is the same on all tested browsers: Chrome, Safari, Firefox, Edge.

Any clue why? Should I file an issue?

Thanks!

**Comment by reidbarber** [comment, marked as answer]
I don't think this is related to it being controlled, but it does seem like incorrect behavior. Feel free to create an issue so this can be investigated further!

**Reply by janvorwerk** [reply]
Thanks for your quick reply. I opened #5770.
There is some relationship with the popover being controlled as you can see in the bug description... don't ask me why 🤔

**Reply by reidbarber** [reply]
Thanks! Interesting, I just mentioned that since I noticed data-pressed stays true even when it isn't controlled.
